app/Database/migrations/2021_11_23_184532_add_type_rating_tables.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use App\Contracts\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddTypeRatingTables extends Migration
+{
+    public function up()
+    {
+        if (!Schema::hasTable('typeratings')) {
```

Comment:
The table name and prefixes should be `type_ratings` and `type_rating_user`, `type_rating_subfleet`, etc. Just keeping with convention.

Reply:
Model name is `Typerating` , this is why the table name uses `typeratings` and intermediates followed that same name.

When I first build this, model name was `TypeRating` and the table names was like you suggested, it caused some problem with relationships (even though I used proper foreign keys). To overcome those problems I removed the camel case naming.

But left the repository name as it was.